### PR TITLE
Using v8_presubmit to check and lint code

### PR DESCRIPTION
**Pull request by luyahan**
Refer to https://logs.chromium.org/logs/v8/buildbucket/cr-buildbucket.appspot.com/8852959421288440864/+/u/Presubmit/execution_details

### CODE REVIEW

**Comment by luyahan** [comment]
Hi @fw-brice . I find that running  `git cl format --presubmit && git diff -- exit-code`  will reutrn cpplint error on upstream' s changes. 
For example on js-heap-broker.cc:
```
#16 [v8-precheck 3/4] RUN git cl format --presubmit && git diff --exit-code
#16 70.07 diff --git a/src/baseline/baseline-compiler.cc b/src/baseline/baseline-compiler.cc
#16 70.07 index ef8e1c5f86..251e2495d1 100644
#16 70.07 --- a/src/baseline/baseline-compiler.cc
#16 70.07 +++ b/src/baseline/baseline-compiler.cc
#16 70.07 @@ -219,7 +219,6 @@ void MoveArgumentsForDescriptor(BaselineAssembler* masm,
#16 70.07  
#16 70.07  }  // namespace detail
#16 70.07  
#16 70.07 -
#16 70.07  BaselineCompiler::BaselineCompiler(
#16 70.07      Isolate* isolate, Handle<SharedFunctionInfo> shared_function_info,
#16 70.07      Handle<BytecodeArray> bytecode)
#16 70.07 diff --git a/src/compiler/js-heap-broker.cc b/src/compiler/js-heap-broker.cc
#16 70.07 index 3145e64da8..aeada588fe 100644
#16 70.07 --- a/src/compiler/js-heap-broker.cc
#16 70.07 +++ b/src/compiler/js-heap-broker.cc
#16 70.07 @@ -1249,7 +1249,7 @@ HeapObjectData::HeapObjectData(JSHeapBroker* broker, ObjectData** storage,
#16 70.07    CHECK_IMPLIES(kind == kSerializedHeapObject,
#16 70.07                  broker->mode() == JSHeapBroker::kSerializing);
#16 70.07    CHECK_IMPLIES(broker->mode() == JSHeapBroker::kSerialized,
#16 70.07 -                    kind == kBackgroundSerializedHeapObject);
#16 70.07 +                kind == kBackgroundSerializedHeapObject);
#16 70.07  }
#16 70.07  
#16 70.07  InstanceType HeapObjectData::GetMapInstanceType() const {
#16 70.07 @@ -2140,12 +2140,12 @@ HEAP_BROKER_BACKGROUND_SERIALIZED_OBJECT_LIST(DEFINE_IS)
#16 70.07  HEAP_BROKER_NEVER_SERIALIZED_OBJECT_LIST(DEFINE_IS)
```

And I find google check code by "tools/v8_presubmit.py" from https://logs.chromium.org/logs/v8/buildbucket/cr-buildbucket.appspot.com/8852959421288440864/+/u/Presubmit/execution_details.
It don't report warning on js-heap-broker.cc.
So should we also  using presubmit to check?

File: Dockerfile [hunk 1]
```diff
@@ -44,7 +44,11 @@ RUN (git remote add riscv https://github.com/${GITHUB_REPOSITORY} && \
 FROM v8-riscv as v8-precheck
 RUN git fetch riscv riscv64
 RUN bash /root/commit-msg-check.sh riscv/riscv64 $(git log --format="%H" -n 1)
-RUN git cl format --presubmit && git diff --exit-code
+RUN apt install python-pip
+RUN pip install coverage
+RUN pip install numpy
+RUN pip install mock
```
Comment (ghost) [inline Dockerfile]: Let's move all of these `pip install` commands up into `v8-base`, after the `apt-get` command.

**Review by ghost — CHANGES_REQUESTED** [review]
This is great! I just have one requested change.

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**